runtime/oti/vmconstantpool.xml

```diff
@@ -62,6 +62,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<classref name="java/lang/LayerInstantiationException" flags="opt_module" versions="9-"/>
 	<classref name="java/lang/reflect/Method" flags="opt_reflect"/>
 	<classref name="java/lang/reflect/Field" flags="opt_reflect"/>
+	<classref name="java/lang/reflect/RecordComponent" flags="opt_reflect"/>
```

Comment:
This needs a a `versions="14-"` tag as well as the class won't exist on earlier releases